packages/grafana-toolkit/src/cli/tasks/plugin.sign.ts

```diff
@@ -1,7 +1,7 @@
 import path from 'path';
 import { buildManifest, signManifest, saveManifest } from '../../plugins/manifest';
 import { Task, TaskRunner } from './task';
-
+import { promises as fs } from 'fs';
```

Comment:
```suggestion
import fs from 'fs/promises';
```

Reply:
@mckn thanks, I made it a bit more precise by importing only the method I need https://github.com/grafana/grafana/pull/44326/commits/0df66b4fe820636ca44483d549e8a43460481246#diff-88a7b4010fb75f53dac5a4ea709f796debb3cfa5b36a63922b398f22b3764e34R7